apps/outlook/public/signout-redirect.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en" data-framework="typescript">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>AgreeTo Microsoft Authentication</title>
+    <script src="https://cdn.tailwindcss.com"></script>
+
+    <script>
+      const params = new Proxy(new URLSearchParams(window.location.search), {
+        get: (searchParams, prop) => searchParams.get(prop),
+      });
+      const token = params.token;
+      localStorage.setItem("token", token);
```

Comment:
Shouldn't we simply `setItem("token", null)`?

Reply:
Yep might as well 👍